packages/pdf/src/RPA/PDF/keywords/model.py

```diff
@@ -0,0 +1,585 @@
+import re
+import sys
+from collections import OrderedDict
+from typing import Any, Iterable
+
+import PyPDF2
+import pdfminer
+from pdfminer.converter import PDFConverter
+from pdfminer.layout import (
+    LAParams,
+    LTPage,
+    LTText,
+    LTTextBox,
+    LTLine,
+    LTRect,
+    LTCurve,
+    LTFigure,
+    LTTextLine,
+    LTTextBoxVertical,
+    LTChar,
+    LTImage,
+    LTTextGroup,
+)
+from pdfminer.pdfdocument import PDFDocument
+from pdfminer.pdfparser import PDFParser
+from pdfminer.utils import enc, bbox2str
+from pdfminer.pdfpage import PDFPage
+from pdfminer.pdfinterp import PDFResourceManager, PDFPageInterpreter
+from pdfminer.converter import PDFConverter
+
+from RPA.PDF.keywords import (
+    LibraryContext,
+    keyword,
+)
+
+
+def iterable_items_to_int(bbox) -> list:
+    if bbox is None:
+        return list()
+    return list(map(int, bbox))
+
+
+class RpaFigure:
```

Comment:
I'm for some reason really bothered by the naming of these model classes. Why are they RPA-specific primitives?